postgres-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/postgres/PostgresDatabaseConnector.java

```diff
@@ -37,7 +37,7 @@ public void refresh() throws ConnectorCheckedException
         try
         {
             List<PostgresDatabase> dbs = sourceDatabase.getDabaseNames();
- /*           List<DatabaseElement> knownDatabases = context.getMyDatabases(1, 100);
```

Comment:
Can we use symbolic constants for the max page size? Potentially in future configuration (this occurs in various other places with constants - I'll just mention it here)

Reply:
Good point.